### ISSUE TITLE
[FEATURE] Support UWP for SkiaSharp 3.0

### ISSUE THREAD

**Issue by tipa** [mono/SkiaSharp#2746]
### Is your feature request related to a problem?

I am using UWP and migrating to WinUI3 is not an option (yet)

### Describe the solution you would like

I would like to use SkiaSharp 3.0 on UWP but when attempting to use it, I get the following error:
```
NU1202: Package SkiaSharp.Views 3.0.0-preview.1.8 is not compatible with uap10.0.17763 (UAP,Version=v10.0.17763) / win10-x86-aot. Package SkiaSharp.Views 3.0.0-preview.1.8 supports:
  - net462 (.NETFramework,Version=v4.6.2)
  - net7.0-android33.0 (.NETCoreApp,Version=v7.0)
  - net7.0-ios16.1 (.NETCoreApp,Version=v7.0)
  - net7.0-maccatalyst16.1 (.NETCoreApp,Version=v7.0)
  - net7.0-macos13.0 (.NETCoreApp,Version=v7.0)
  - net7.0-tizen7.0 (.NETCoreApp,Version=v7.0)
  - net7.0-tvos16.1 (.NETCoreApp,Version=v7.0)
  - net7.0-windows10.0.19041 (.NETCoreApp,Version=v7.0)
```

As announced [here](https://github.com/mono/SkiaSharp/blob/main/changelogs/SkiaSharp/3.0.0/SkiaSharp.humanreadable.md), v3.0 should be compartible with .NET Standard 2.0 (which is what UWP is using) so I am not sure if this is a bug

### Describe alternatives you have considered

Migrating to WinUI3 is not an option as it has bad performance, unsatisfying bundle size and various other issues.
I could also just stay on SkaiSharp 2.x, but I'd prefer to use the "latest and greatest" SkiaSharp version ;)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mattleibow** [comment]
I heard echos of the case where win32 dlls now work on UWP - especially so we can build using Clang.

For now, you could try copying in the view source from the 2.x branch and see if that works. A bit of work, but if that is the case then maybe we can use the win32 dlls from 3.x and then just have a small uwp dll for the views.

**Comment by mattleibow** [comment]
I know this is something I have heard from @dotMorten, so I guess I will have to start by reading my old chats and inboxes with all the steps. But, if anyone else has had success, then we might be able to do this easily. I do know the skia codebase is optimized for and supported on Clang. MSVC is a little dodgy in areas as they have slow code paths and sometimes build errors.

**Comment by dotMorten** [comment]
Correct we no longer build Skia specifically for uwp but use standard win32 dlls in our uwp apps and they still pass app certification

**Comment by mattleibow** [comment]
I was typing steps, but the replies are too fast. Very good news.

**Comment by dotMorten** [comment]
You just need to put the native libs in the runtimes/win-xXX/native folder and both net6 and UWP will deploy them correctly. The managed api to call into them will be the same too but you might need a uwp specific build of the .net assembly but that's easy to do with multi targeting.
Once done build a uwp app using it and run it through the app certification kit just to make sure there are no cert issues with banned api calls

**Comment by mattleibow** [comment]
Thanks!

**Comment by jackbond** [comment]
I realize the preview was only published a week ago, but do you guys have a ballpark goal for the 3.0 release? Thanks. And thanks for the great library!

**Comment by charlesroddie** [comment]
> WinUI3 ... bad performance, unsatisfying bundle size

I agree and the fix for both will be in NativeAOT support and associated work. This doesn't look far off and is being worked on in https://github.com/microsoft/CsWinRT/tree/staging/AOT . If we are lucky we may get both goodies at the same time (usable WinUI3, SkiaSharp 3.0) and this issue will be avoided.

**Comment by charlesroddie** [comment]
Does this project become easier if the project is to support the just-announced dotnet9 UWP? https://devblogs.microsoft.com/ifdef-windows/preview-uwp-support-for-dotnet-9-native-aot/

**Comment by FoggyFinder** [comment]
> Ah, I wonder if this is because you are using the wasdk build of the interop and angle.

same result with both 

```cs
Build(arch, "libEGL", wasdk: false);
```

and

```cs
Build(arch, "libEGL", wasdk: true);
```

for angle.

----

additional info: `Egl.eglGetError()` returns `EGL_BAD_SURFACE`

**Comment by mfeingol** [comment]
Just wondering... is there any ongoing work on this?
